happi/containers.py

```diff
@@ -297,3 +297,43 @@ class Motor(Device):
     device_class.default = 'pcdsdevices.device_types.Motor'
     system = copy(Device.system)
     system.default = 'motion'
+
+
+class Camera(Device):
+    """
+    A Generic EpicsCamera
+    """
+    device_class = copy(Device.device_class)
+    device_class.default = 'pcdsdevices.device_types.Camera'
+    system = copy(Device.system)
+    system.default = 'cameras'
+
+
+class AreaDetector(Device):
+    """
+    A Generic EpicsAreaDetector
+    """
+    device_class = copy(Device.device_class)
+    device_class.default = 'pcdsdevices.device_types.AreaDetector'
+    system = copy(Device.system)
+    system.default = 'detectors'
+
+
+class Acromag(Device):
+    """
+    A Generic class for Acromag
+    """
+    device_class = copy(Device.device_class)
+    device_class.default = 'pcdsdevices.device_types.Acromag'
+    system = copy(Device.system)
+    system.default = 'acromag'
+
+
+class ControlsTriggers(Device):
+    """
+    A Generic class for Controls Triggers
+    """
+    device_class = copy(Device.device_class)
+    device_class.default = 'pcdsdevices.device_types.ControlsTriggers'
```

Comment:
These have to actually exist in the `pcdsdevices` module. I think this is called `Trigger` or something?